ListDetail/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<resources>
+    <string name="list_details_app_name">Listendetail</string>
+    <string name="list_details_camera">Kamera</string>
+    <string name="list_details_device">Gerät</string>
+    <string name="list_details_device_name">Oberflächen-Duo</string>
+    <string name="list_details_image_spec">1: 2,0 1/15 2,51 mm ISO 520</string>
```

Comment:
Not sure if we should translate this, at least the first part `1: 2,0" differs with what we have in English and I think is wrong.